kratos/processes/entity_erase_process.cpp

```diff
@@ -0,0 +1,162 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/kratos_flags.h"
+#include "processes/entity_erase_process.h"
+#include "utilities/variable_utils.h"
+#include "utilities/auxiliar_model_part_utilities.h"
+
+namespace Kratos
+{
+template<class TEntity>
+const Kratos::Flags EntitiesEraseProcess<TEntity>::ERASE_ALL_ENTITIES(Kratos::Flags::Create(0));
+template<class TEntity>
+const Kratos::Flags EntitiesEraseProcess<TEntity>::NOT_ERASE_ALL_ENTITIES(Kratos::Flags::Create(0, false));
+template<class TEntity>
+const Kratos::Flags EntitiesEraseProcess<TEntity>::ASSIGN_FLAG(Kratos::Flags::Create(1));
```

Comment:
Why we have still this flag? I don't know if I can trust on GitHub and this is an old version....

Reply:
![imagen](https://user-images.githubusercontent.com/19991680/56724064-8f3ea900-674a-11e9-855a-0bf42251a7c5.png)
We agreed on REMOVE_ALL_ENTITITES or ERASE_ALL_ENTITIES

Reply:
Yes, but what is the flag 1 just about your message. The old ASSIGN_FLAG....

Reply:
What do you mean?

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/548f7c5cba9e6a5cc3fc88c3d833062b18aa2497/kratos/processes/entity_erase_process.cpp#L30

I mean why you still have this flag defined if in option you are using `ERASE_ALL_ENTITIES`